exercises/protein-translation/canonical-data.json

```diff
@@ -136,13 +136,13 @@
           "expected": ["Methionine","Phenylalanine"]
         },
         {
-          "description": "Translation stops if STOP codon in middle of sequence of three-codon sequence",
+          "description": "Translation stops if STOP codon in middle of three-codon sequence",
           "property": "proteins",
           "strand": "UGGUAGUGG",
           "expected": ["Tryptophan"]
         },
         {
-          "description": "Translation stops if codon in middle of sequence of six-codon sequence",
+          "description": "Translation stops if codon in middle of six-codon sequence",
```

Comment:
Shouldn't "Translation stops if codon" be "Translation stops if STOP codon"

Reply:
That is indeed the case - good spot! I'll fix that one too :slightly_smiling_face: